src/components/Hero/desktop.styles.ts

```diff
@@ -0,0 +1,93 @@
+import { theme } from '@/theme';
+import { getTaglineIconBase, ICON_MIN_SIZE_DESKTOP } from './icon.utils';
+import type { HeroStyle, HeroStylesStructure } from './shared.styles';
+
+export const desktopOverrides: Partial<HeroStylesStructure> = {
+  container: {
+    padding: `clamp(${theme.spacing.md}, 5vw, calc(${theme.spacing.xl} * 2))`,
+  },
+
+  content: {
+    maxWidth: 'clamp(90%, 80vw, 1400px)',
+    gap: `clamp(${theme.spacing.md}, 4vw, calc(${theme.spacing.xl} * 2))`,
+    padding: `0 clamp(${theme.spacing.xs}, 2vw, ${theme.spacing.xl})`,
+  },
+
+  titleContainer: {
+    gap: `clamp(${theme.spacing.xs}, 1vw, ${theme.spacing.md})`,
+  },
+
+  title: {
+    fontSize: 'clamp(4rem, 10vw, 7rem)',
```

Comment:
Do not hardcoded anything that we can bring from the theme object, font sizes, gaps etc...

This comment is relevant to all of this file